Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7.9-slim-buster as base
+
+ENV PATH /usr/local/bin:$PATH
+ENV VIRTUAL_ENV "/venv"
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH "$VIRTUAL_ENV/bin:$PATH"
+WORKDIR /app
+RUN pip install poetry
+EXPOSE 5000
+COPY poetry.lock pyproject.toml /app/
```

Comment:
This `pyproject.toml` file seems to be missing dependencies for pytest & gunicorn, which causes errors like that you're seeing on Travis.